addons/kured/resources/foo.yml

```diff
@@ -0,0 +1,15 @@
+kind: Role
```

Comment:
These files probably don't belong to this pr? 😄

Reply:
Definitely not. They look like some leftovers from PSP testing. I'll remove them

Reply:
done